playground/AWS/AWS.AppHost/AWS.AppHost.csproj

```diff
@@ -13,10 +13,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\src\Aspire.Dashboard\Aspire.Dashboard.csproj" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.AWS\Aspire.Hosting.AWS.csproj" IsAspireProjectResource="false" />    
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.AppHost\Aspire.Hosting.AppHost.csproj" IsAspireProjectResource="false" />
-    
+    <ProjectReference Include="..\..\..\src\Aspire.Dashboard\Aspire.Dashboard.csproj" Condition="'$(BuildForTest)' != 'true'" />
+    <AspireProjectOrPackageReference Include="Aspire.Hosting.AWS" IsAspireProjectResource="false" />
```

Comment:
Similar comment here for `IsAspireProjectResource="false"`. We should only need this when it is a `ProjectReference`. So when we create the `AspireProjectOrPackageReference` => `ProjectReference`, we should be able to inject this metadata.